### ISSUE TITLE
BibIngest: new module to cover the needs of persistent storage of submitted material (ingestion packages) to the repository.

### ISSUE THREAD

**Issue by invenio-developers** [inveniosoftware/invenio#1289]
Originally by _[nkasioum (@kasioumis)](https://github.com/kasioumis)_ on 2013-02-07

The [BibIngest](http://invenio-software.org/wiki/Development/Modules/BibIngest) module:
- can be useful for anyone who needs to store submitted material in its original format and wants to be able to retrieve it for later extraction of information, resubmission, and dissemination.
- should also cover the requirements of the Open Archival Information System (OAIS) specification for the acquisition and storage of the Submission Information Package (SIP) ("an Information Package that is delivered by the Producer to the OAIS [Preservation System]") for use in the construction of one or more Archival Information Packages (AIP).

Development: [master-bibingest](http://invenio-software.org/repo/personal/invenio-nkasioum/log/?h=master-bibingest) branch.

**Comment by kaplun** [comment]
Is there an equivalent work/idea for the new `master`?

**Comment by tiborsimko** [comment]
Yes, early SIP, richer AIP, memento-friendly DIP.  Coming after current module re-factoring efforts. See e.g. https://github.com/inveniosoftware/invenio/pull/3364#issuecomment-121654459.  Already in the manpower work plan.
